Userland/Services/SpiceAgent/main.cpp

```diff
@@ -20,6 +21,11 @@ static constexpr auto SPICE_DEVICE = "/dev/hvc0p1"sv;
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
+    if (!FileSystem::exists(SPICE_DEVICE)) {
+        dbgln("{} doesn't exist... exiting!", SPICE_DEVICE);
+        return -1;
```

Comment:
Why don't we return a meaningful `AK::Error` here :^)?

Reply:
`String::formatted` is flaky with `Error` from what I can see, is a "Failed to find spice device file!" sufficient for this?

Reply:
`String::formatted` gives you a `String` which hold the information, whereas `Error` can only hold a `StringView`. If you connect both, you will have some lifetime issue.

So yeah, that's good enough.